logstash-integration-elastic_enterprise_search.gemspec

```
@@ -24,11 +24,10 @@ Gem::Specification.new do |s|
   }
 
   # Gem dependencies
+  s.add_runtime_dependency "manticore", '>= 0.8.0', '< 1.0.0'
```

I think we could use 

```suggestion
  s.add_runtime_dependency "manticore", '~> 0.8'
```

to have the same semantic